hookplugins/CriPlugin.go

```diff
@@ -0,0 +1,24 @@
+package hookplugins
+
+import (
+	"github.com/alibaba/pouch/apis/types"
+)
+
+// CriPlugin defines places where a plugin will be triggered in CRI api lifecycle
+type CriPlugin interface {
+	// PreCreateContainer defines plugin point where receives a container create request, in this plugin point user
+	// could the container's config in cri interface.
```

Comment:
```suggestion
	// could update the container's config in cri interface.
```

or something else? :)